pkg/kubelet/status/status_manager.go

```diff
@@ -675,6 +675,18 @@ func (m *manager) syncPod(uid types.UID, status versionedPodStatus) {
 			"pod", klog.KObj(pod),
 			"oldPodUID", uid,
 			"podUID", translatedUID)
+		if kubetypes.IsMirrorPod(pod) {
+			// if the mirror pod for its static pod has not received by pod manager(at kubelet start time),
+			// we should not delete static pod status immediately, otherwise we may lose initial pod status like
+			// UnexpectedAdmissionError
+			staticPod, ok := m.podManager.GetPodByUID(uid)
+			if ok {
+				_, ok := m.podManager.GetMirrorPodByPod(staticPod)
+				if !ok {
+					return
```

Comment:
I think the log message is needed here different from one on line 674. I think differentiating the delete status and skip update and just skip update is important for troubleshooting

Reply:
Make sense, thanks Sergey